airflow/contrib/operators/azure_container_instances_operator.py

```diff
@@ -104,11 +99,9 @@ class AzureContainerInstancesOperator(BaseOperator):
             )
     """
 
-    template_fields = ('name', 'environment_variables')
-
     @apply_defaults
     def __init__(self, ci_conn_id, registry_conn_id, resource_group, name, image, region,
-                 environment_variables=None, volumes=None, memory_in_gb=None, cpu=None,
```

Comment:
In the original PR for this file, it was requested the None be used as default params:

https://github.com/apache/airflow/pull/4121
https://graysonkoonce.com/always-use-none-for-default-args-in-python/

Reply:
Will put Nones back in.